theano/sandbox/cuda/cuda_ndarray.cu

```diff
@@ -3244,10 +3244,10 @@ CudaNdarray_gpu_init(PyObject* _unused, PyObject* args)
         if (cnmem > 1)
             mem = cnmem * 1024 * 1024;
         else{
-            // Clip to 98% to let memory for the driver.
-            // 98.5% didn't worked in some cases.
-            if (cnmem > .98){
-                cnmem = .98;
+            // Clip to 95% to let memory for the driver.
+            // 98% didn't worked in some cases.
+            if (cnmem > .99){
```

Comment:
It can still grow below 95% if needed, right? In that case, I think capping at 95% would be OK.
And if people want to hack their way, then they can use an absolute size in MB.